### PR TITLE
Use custom JSON encoder

### LINKED ISSUE #55: object of type time is not JSON serializable
example: staging
Croix rouge --> table RESIDANT_PLAN_ANI

### PR DESCRIPTION
**Pull request by simonvadee**
Fix #55

### CODE REVIEW

File: pagai/json_encoder.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import flask
+
+
+class MyJSONEncoder(flask.json.JSONEncoder):
+    """
+    We make our JSONEncoder to override the default method.
+    Decimal values are not handled by Flask's JSONEncoder so
+    we turn them to strings.
+    This doesn't cause any type problem because the transformer will turn
+    them back to numerical values if needed (depending on the FHIR element type).
```
Comment (Jasopaum) [inline pagai/json_encoder.py:12]: You can remove this part of the docstring

File: pagai/json_encoder.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import datetime
+import decimal
+import flask
+
+
+class MyJSONEncoder(flask.json.JSONEncoder):
+    """
+    We make our JSONEncoder to override the default method.
+    Decimal values are not handled by Flask's JSONEncoder so
```
Comment (Jasopaum) [inline pagai/json_encoder.py:9]: Hum, I know it's copied from fhir-river but maybe we should speak about all the case we handle or none of them, what do you think?

File: pagai/json_encoder.py (hunk 1 above)
Comment (simonvadee) [reply pagai/json_encoder.py:12]: thx, done.

File: pagai/json_encoder.py (hunk 2 above)
Comment (simonvadee) [reply pagai/json_encoder.py]: agreed, done.

File: pagai/json_encoder.py [hunk 3]
```diff
@@ -6,10 +6,10 @@
 class MyJSONEncoder(flask.json.JSONEncoder):
     """
     We make our JSONEncoder to override the default method.
-    Decimal values are not handled by Flask's JSONEncoder so
-    we turn them to strings.
-    This doesn't cause any type problem because the transformer will turn
-    them back to numerical values if needed (depending on the FHIR element type).
+    When jsonifying the rows retrieved by SQLAlchemy, we need to handle type
+    serialization manually when flask does not handle them already.
+    - SQLALchemy parses DB "number" objects to decimal.Decimal so we convert theml to floats
```
Comment (Jasopaum) [inline pagai/json_encoder.py:11]: typo: `theml` -> `them`

**Review by Jasopaum — DISMISSED** [review]

**Review by Jasopaum — APPROVED** [review]

**Pull request merged**